app/models/user.rb

```diff
@@ -1,5 +1,9 @@
 class User < ApplicationRecord
-  has_many :posts
-  has_many :comments
-  has_many :likes
+  has_many :posts, dependent: :destroy
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
```

Comment:
- [ ]  This is a nice approach but it's recommended to use `foreign_key: 'author_id'`

Reply:
:+1: This comment is addressed successfully :heavy_check_mark: